pkg/service/helper.go

```diff
@@ -32,3 +32,22 @@ func diskAttachmentByVmAndDisk(connection *ovirtsdk.Connection, vmId string, dis
 		diskId: diskId,
 	}
 }
+
+func diskAttachmentByDisk(connection *ovirtsdk.Connection, diskId string) (*ovirtsdk.DiskAttachment, *ovirtsdk.DiskAttachmentService, error) {
+	vmsService := connection.SystemService().VmsService()
+	for _, vm := range vmsService.List().MustSend().MustVms().Slice() {
+		vmService := connection.SystemService().VmsService().VmService(vm.MustId())
+		attachments, err := vmService.DiskAttachmentsService().List().Send()
+		if err != nil {
+			return nil, nil, err
+		}
+
+		for _, attachment := range attachments.MustAttachments().Slice() {
+			if attachment.MustId() == diskId {
```

Comment:
@bennyz shouldn't this be:
```
if attachment.MustDisk().MustId() == diskId {
```
Like https://github.com/openshift/ovirt-csi-driver/blob/master/pkg/service/helper.go#L26
?

Reply:
I think they are equivalent, we don't have separate a ID for the attachment, so this version seems better